README.md

```diff
@@ -1,13 +1,15 @@
 # chapi [![Build Status](https://travis-ci.org/msiebeneicher/chapi.svg?branch=master)](http://travis-ci.org/msiebeneicher/chapi) [![Scrutinizer Code Quality](https://scrutinizer-ci.com/g/msiebeneicher/chapi/badges/quality-score.png?b=master)](https://scrutinizer-ci.com/g/msiebeneicher/chapi/?branch=master) [![Code Coverage](https://scrutinizer-ci.com/g/msiebeneicher/chapi/badges/coverage.png?b=master)](https://scrutinizer-ci.com/g/msiebeneicher/chapi/?branch=master)
-Chronos api client for your console
+Chronos and marathon api client for your console. 
 
 ## Description
-Manage your [Chronos](https://github.com/mesos/chronos) jobs like a git repository on your console:
+Manage your [Chronos](https://github.com/mesos/chronos) and [Marathon](https://github.com/mesosphere/marathon) jobs like a git repository on your console:
```

Comment:
The correct term is `tasks`, not jobs.
See https://github.com/mesosphere/marathon
> The state of running tasks gets stored in the Mesos state abstraction.

In general, on Mesos you will launch tasks. Nitpicking, i know :)